### PR TITLE
add helper script to make releasing easier

### PR DESCRIPTION
**Pull request by phoracek**
Make releases easier.

### CODE REVIEW

**Comment by phoracek** [comment]
@SchSeba @booxter

File: hack/release.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+echo 'Commit Makefile with bumped versions'
+git add Makefile
+
+echo 'Commit updated Makefile'
+git commit -s -m "release ${released_version} - bump versions in Makefile"
+
+echo 'Push changes to forked repo'
+git push ${fork_remote}
+
+echo 'The rest is on you, open a new PR. Once the PR is merged, do not forget to tag the first of these commits, include change log and upload released manifests'
```
Comment (SchSeba) [inline hack/release.sh]: maybe change this to tag the commit started with `Merge pull request <pr-number>` ?

Comment (phoracek) [reply hack/release.sh]: I know it is a little weird, but release should be on the first one, where we generate manifests for the new version. The second commit is starting next version.

Practically there shouldn't be a difference. If you think it makes more sense to keep it on top of the PR, I will change it.

**Comment by phoracek** [comment]
/hold

I need to set container tag too.

**Pull request merged**